src/main/java/uk/gov/justice/hmpps/prison/aop/connectionproxy/OracleConnectionAspect.kt

```diff
@@ -40,7 +40,7 @@ class OracleConnectionAspect(
 
   @Throws(SQLException::class)
   private fun Connection.openProxySessionConnection(): Connection {
-    log.info("Configuring Oracle Proxy Session for NOMIS user {}", this)
```

Comment:
This was actually logging the object id of the `Connection` instance - no value that I can see, we just want to know which kind of connection this request is using.